### PR TITLE
Course show in resource

### PR DESCRIPTION
**Pull request by hungph-dev-ict**

### CODE REVIEW

**Comment by hungph-dev-ict** [comment]
Register course for student

**Comment by hungph-dev-ict** [comment]
ready

File: app/Http/Controllers/CoursesUserController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public function __construct(CoursesUser $coursesUser)
+    {
+        $this->modelCoursesUser = $coursesUser;
+    }
+
+    public function changeStatus(Request $request)
+    {
+        $data = $request->all();
+        $result = $this->modelCoursesUser->changeCourseUser($data);
+        // dd($result);
```
Comment (BunnyPi04) [inline app/Http/Controllers/CoursesUserController.php]: xóa dd() đi nhé

File: resources/views/courses/show.blade.php [hunk 2]
```diff
@@ -233,5 +248,51 @@
...
+                        },{
+                            // settings
+                            type: 'success'
+                        });
+                    }                  
+                }
+            );
+        })
+    </script>
 @endsection
```
Comment (BunnyPi04) [inline resources/views/courses/show.blade.php]: thiếu dòng trắng cuối file

File: resources/views/courses/show.blade.php [hunk 3]
```diff
@@ -233,5 +248,51 @@
...
+                            type: 'warning'
+                        });
+                    } else {
+                        $('#active-btn button').removeClass('btn-primary');
+                        $('#active-btn button').addClass('btn-warning');
+                        $('#active-btn button').text(' {{ __('cancel_active') }} ');
+
+                        $.notify({
+                            // options
+                            message: 'Active course successfully, wait approve from your teacher.'
```
Comment (BunnyPi04) [inline resources/views/courses/show.blade.php]: i18n

**Comment by BunnyPi04** [comment]
commented

**Comment by hungph-dev-ict** [comment]
ready

**Pull request merged**